src/video-player.js

```diff
@@ -360,7 +360,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
     this.initQualitySelector = () => {
       if (_options.qualitySelector !== false) {
-        this.videojs.httpSourceSelector({ default: 'high' });
+        this.videojs.httpSourceSelector({ default: 'Auto' });
```

Comment:
@yaniv-li should this be 'auto' or 'Auto'? [see readme](https://www.npmjs.com/package/videojs-http-source-selector#auto)
either way, I don't see any change when playing with this value

Reply:
I think we should re-implement this according to what our design

Reply:
show be lower case i think